modules/heat_conduction/include/constraints/ModularGapConductanceConstraint.h

```diff
@@ -10,6 +10,7 @@
 #pragma once
 
 #include "ADMortarConstraint.h"
+#include "ModularGapConductanceConstraint.h"
```

Comment:
Circular

Reply:
You just need a forward, I think

```suggestion
class GapFluxModelBase;
```